libs/vre/shared/app-state/src/lib/user/user.selectors.ts

```diff
@@ -11,6 +11,11 @@ export class UserSelectors {
     return state.isLoading;
   }
 
+  @Selector([UserState])
+  static usersLoading(state: UserStateModel): boolean | undefined {
```

Comment:
only boolean

Reply:
Indeed not needed. Copy/pasted it from user loading state.